enhancements/sdk-migrate-bundle.md

```diff
@@ -0,0 +1,327 @@
+---
+title: sdk-migrate-bundle
+authors:
+  - "@jmrodri"
+reviewers:
+  - "@estroz"
+  - "@ecordell"
+  - "@rashmigottipati"
+approvers:
+  - "@estroz"
+  - "@rashmigottipati"
+creation-date: 2021-02-08
+last-updated: 2021-02-08
+status: implementable
+see-also:
+replaces:
+superseded-by:
+---
+
+# Migrate Bundle
+
+## Release Signoff Checklist
+
+- [X] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+
+## Open Questions [optional]
+
+1. The bundles we build today are done via `docker` from the project's
+   `Makefile` where users can adapt their image builder to whatever they want.
+   Do we want to allow this from the CLI?
+
+## Summary
+
+Add a subcommand to the Operator SDK CLI that will allow users to migrate
+their operator's packagemanifest format to the new bundle format.
+
+## Motivation
+
+There are operators that exist that are still using the packagemanifest
+format. This subcommand will allow them to migrate these packagemanifests to
+bundles which is the preferred OLM packaging format.
+
+### Goals
+
+- Provide a subcommand to migrate one packagemanifest to a set of bundles.
```

Comment:
What will happen to the existing command in SDK for deploying an operator packagemanifests using OLM? Do we still want to support that and will it stay intact? 

Maybe worth mentioning and clarifying that point somewhere in this EP.